.github/RELEASING.md

```diff
@@ -1 +1,16 @@
-# TODO
+# How to Release
+
+This project uses lerna to do versioning. This makes releasing really easy and changelogs are automatically generated.
```

Comment:
Inspired by [emotion](https://github.com/emotion-js/emotion/blob/master/RELEASING.md)

Reply:
Just a little suggestion about this phrase here: `This makes the process of generate a new release and changelogs automatically.`